include/deal.II/base/vectorization.h

```diff
@@ -4110,6 +4110,27 @@ namespace std
 
 
 
+  /**
+   * Variant of above function that takes an integer as second argument.
+   *
+   * @relatesalso VectorizedArray
+   */
+  template <typename Number, int width>
+  inline ::dealii::VectorizedArray<Number, width>
+  pow(const ::dealii::VectorizedArray<Number, width> &x, const int p)
+  {
+    Number values[::dealii::VectorizedArray<Number, width>::n_array_elements];
+    for (unsigned int i = 0;
+         i < dealii::VectorizedArray<Number, width>::n_array_elements;
+         ++i)
+      values[i] = std::pow(x[i], p);
```

Comment:
Are there no intrinsics that can do all of this at once?

Reply:
@bangerth There are no portable intrinsics that can do all of this at once. We are currently investigating more efficient implementations of `std::pow` for `VectorizedArray` as well.
For the time being, I'd really like to see this variant merged.

Reply:
Thanks for the confirmation!